src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -50,7 +50,7 @@ def send_incident_alert_notification(
         logger.info("rule.fail.slack_post", extra={"error": str(e)})
 
 
-def get_referrer_qstring(notification: BaseNotification, recipient: Team | User) -> str:
+def get_referrer_qstring(notification: OrganizationNotification, recipient: Team | User) -> str:
```

Comment:
@mgaeta Not sure why you're using a more narrow type here, wouldn't `BaseNotification` work here? I think these functions work just fine for all types of notifications.